test/docx/body-reader.tests.js

```diff
@@ -861,6 +873,23 @@ function createLinkedBlip(relationshipId) {
     return new XmlElement("a:blip", {"r:link": relationshipId});
 }
 
+function createHyperlink(link) {
```

Comment:
I'd also limit this function to just creating the XML for the test case, and leaving the reading to the actual test case itself for clarity.

Also, knowing to pluck out the third child feels a bit magic.